src/PaperG/ScenarioBlob/Blobs/UnmanagedFacebook/FacebookAdSet.php

```diff
@@ -166,4 +166,4 @@ private function validateGoal($validOptimizationGoals)
 
         return in_array($this->optimizationGoal, $validOptimizationGoals);
     }
-} 
+}
```

Comment:
We can move what we can for now without impacting backwards compatibility and then map out what we'd like after.

There is the issue where a scenario is a combination of several things and these types of datasets don't necessarily have to fit narrowly into single scenarios.  A facebookAdSet might fit into something like \ScenarioBlob\FacebookData